### ISSUE TITLE
User Review Test Token Dashboard

### ISSUE THREAD

**Issue by nahuus123** [keep-network/keep-core#2053]
User experience review Keep testnet dashboard (https://dashboard.test.keep.network/)

**•	Signing in:**
o	When not logged into MetaMask yet: Clicking the MetaMask option gives the 'Connect wallet' pop-up, and a MetaMask screen will indeed open to log in.
o	When already logged into MetaMask: Still this same Connect Wallet screen pops-up, however there is no MetaMask screen as it’s already logged in. Better to also leave out the Connect Wallet pop-up?
o	When refreshing or clicking the logo on the top left it automatically signs out and requires to re-select signin method. I would prefer to be kept signed in and have a sign-out option.
 

**•	General - Menu’s**: Clicking for example the ‘Tokens’ option leads to the same page as Tokens – Overview. For me it would be more clear to have 1 main option ‘Token Overview’ and have only 2 subs. Delegate and Grants. Same for Applications  Applications overview

•	**Page 'Operations'**: There's an adress behind 'Owner'. This is a different adress then the users adress, and It's not clear to me what owner means? Is this related to the contract where the tokens are staked?

•	**Page 'Rewards'**: opening the main Rewards menu leads to the Random Beacon rewards page. Better to have it link to a rewards overview page where you can click the subs?

•	**Page 'Rewards'** – Random Beacon and tBTC. For the history, would be nice to add the authorize date?

•	**Page 'Applications'** – tBTC. Add ETH for bonding section.
o	Maybe also display the beneficiary address, so : NOTE: Withdrawn ETH will go to the beneficiary address: 0x…..
o	 Change ‘Available ETH’ to something more clear like ‘staked ETH’ or ETH available for bonding? Maybe also display here the remaining ETH wallet balance?
•	**Page 'Applications'** – Random Beacon and tBTC 
o	Have the same lay-out & columns. Beacon starts with Details and has a column ‘operator'. tBTC starts with ‘Operator Adress’.  
o	Re-name ‘details’ to ‘status’ and add it to both pages on the last column.
 	 

•	**Page 'Glossary'** – Contents menu. Hovering over them with the mouse does not change to a ‘hand’ icon like it does for the other menus.

•	**Page 'Tokens** 
o	Clicking the links Granted or Owned tokens. If you were already on the 'delegate tokens page' and hovered a bit down, clicking these links will jump to somewhere in the page. I would prefer to always jump to the top of the 'delegate tokens page'

•	**Page 'Tokens** - delegate Tokens
o	Token amount – Limit this field to the actual balance in the grant or wallet.
o	In the ‘Grant Delegation overview’ the 'status' column mentions a date. Should the column name be ‘stake date’?
o	If possible make the descriptions in Authorizer, Operator and Beneficiary Address a bit more self-explanatory? For me it’s not 100% clear at the moment (could be just me).
o	Grant Details section – this is taking up quite a bit of space on the page. This could be smaller that gives a better overview. Also I feel the Unlocked, released, staked data can be a bit confusing. Not sure what the difference is between unlocked and released for example, so maybe add a '?'
o	Grant Details section: On the Unlocked and Staked area it mentions 300,000 of 300K total. Not sure how this will work when you have for example 299,999 staked. Will it do a rounding? May be better to have both as 300,000 of 300,000.

**Comment by liz-shinn** [comment]
Thanks for contributing this! Reviewing this feedback now 👁️ 👁️

**Comment by liz-shinn** [comment]
>When refreshing or clicking the logo on the top left it automatically signs out and requires to re-select signin method. I would prefer to be kept signed in and have a sign-out option.

Agreed, had also noticed this; wasn't sure if this was just on my end. Noted! Will make an issue for this.

**Comment by liz-shinn** [comment]
Cards created: 
1. https://github.com/keep-network/keep-core/issues/2057
2. https://github.com/keep-network/keep-core/issues/2081

>Clicking for example the ‘Tokens’ option leads to the same page as Tokens – Overview. For me it would be more clear to have 1 main option ‘Token Overview’ and have only 2 subs. Delegate and Grants. Same for Applications change to 'Applications overview'
3. Addressing this feedback in: https://github.com/keep-network/keep-core/issues/1999 and https://github.com/keep-network/keep-core/issues/1998

>There's an adress behind 'Owner'. This is a different adress then the users adress, and It's not clear to me what owner means? Is this related to the contract where the tokens are staked?
4. Yup, agreed this is confusing. Addressing this feedback here: https://github.com/keep-network/keep-core/issues/1969

>Grant Details section – this is taking up quite a bit of space on the page. This could be smaller that gives a better overview. Also I feel the Unlocked, released, staked data can be a bit confusing. Not sure what the difference is between unlocked and released for example, so maybe add a '?'
5. Addressing this feedback here: https://github.com/keep-network/keep-core/issues/2003

Thanks @nahuus123 !

**Comment by lionakhnazarov** [comment]
Closing as stale. Time-boxed community usability survey/review for the legacy KEEP token dashboard; the survey window closed years ago and the dashboard has moved to its own repo.
